e2eTests/e2eTestPrompts.md

```diff
@@ -328,3 +328,16 @@ This file contains prompts used for end-to-end testing to ensure each tool is in
 | Tool Name | Test Prompt |
 |:----------|:----------|
 | azmcp-bicepschema-get | How can I use Bicep to create an Azure OpenAI service? |
+
+## Deploy
+| Tool Name | Test Prompt |
+|:----------|:----------|
+| azmcp-deploy-plan-get | Create a plan to deploy this application to azure |
+| azmcp-deploy-iac-rules-get | Show me the rules to generate bicep scripts  |
```

Comment:
WE have a bicep schema tool.  Can this be merged there?  What value add does this give over that?